jazz_test-lambda/config/dev-config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "configKey": "ConfigValueProd"

```

Comment:
ConfigValueDev?

Reply:
Just get rid of these. We don't need any config for this service.

Reply:
removed